server/routes/journeys/update-investigation/evidence-secured/controller.ts

```diff
@@ -0,0 +1,37 @@
+import { NextFunction, Request, Response } from 'express'
+import { SchemaType } from '../../record-investigation/evidence-secured/schemas'
+import { MESSAGE_INVESTIGATION_UPDATED, PatchInvestigationController } from '../../base/patchInvestigationController'
+import { getMaxCharsAndThresholdForAppend, getTextForApiSubmission } from '../../../../utils/appendFieldUtils'
+
+export class UpdateEvidenceSecuredController extends PatchInvestigationController {
+  GET = async (req: Request, res: Response) => {
+    req.journeyData.isUpdate = true
+    const currentEvidenceSecured = req.journeyData.csipRecord!.referral.investigation!.evidenceSecured
+    res.render('record-investigation/evidence-secured/view', {
+      currentEvidenceSecured,
+      evidenceSecured: res.locals.formResponses?.['evidenceSecured'],
+      isUpdate: true,
+      recordUuid: req.journeyData.csipRecord!.recordUuid,
+      ...getMaxCharsAndThresholdForAppend(res.locals.user.displayName, currentEvidenceSecured),
+    })
```

Comment:
`req.journeyData.isUpdate = true` is already set by `/update-investigation/controller` and isn't needed here.

we need a back link on this screen according to the ticket a/c